src/common/image_cache.c

```diff
@@ -50,9 +50,9 @@ void dt_image_cache_allocate(void *data,
       "       longitude, latitude, altitude, color_matrix, colorspace, version,"
       "       raw_black, raw_maximum, aspect_ratio, exposure_bias,"
       "       import_timestamp, change_timestamp, export_timestamp, print_timestamp,"
-      "       output_width, output_height"
-      "  FROM main.images"
-      "  WHERE id = ?1",
+      "       output_width, output_height, cm.maker, cm.model, cm.alias"
+      "  FROM main.images AS mi, main.cameras AS cm"
+      "  WHERE mi.id = ?1 AND cm.id = mi.camera_id",
```

Comment:
We have issue with the CI. I did used 3 SELECT above (for maker/model/lens) as if there is no lens_id found (can be the case early when importing new images) then the main SELECT returns no column. With the 3 SELECT I get NULL for the column which is OK during first run.

I bet that if you try to import new picture with this PR you'll get a lock as shown by the CI.